### PR TITLE
Fix Numerous Spelling, Style, English Issues

### PR DESCRIPTION
**Pull request by waprin**
A 30 min quick, but slow and deliberate, review of the existing codebase.

- Multiple typos and misspellings 
- Multiple missing or incorrect punctuation
- Unused variables

The library is also wildly inconsistent  on its usage of % vs the format function for  string interpolation. I suggest format, but that belongs in its own PR.

http://stackoverflow.com/questions/12382719/python-way-of-printing-with-format-or-percent-form

### CODE REVIEW

**Comment by waprin** [comment]
@dhermes @tseaver

**Comment by dhermes** [comment]
Cool to accept typo / grammar fixes but I'm not sure when punctuation is warranted / if we want to have an opinion on that. Similar on `%` vs. `format`. I used to be all `format` all the time but found it's hard to adhere to in practice (e.g. `logging.log()` assumes `%` formatting)

**Comment by jonparrott** [comment]
> hard to adhere to in practice  (e.g. logging.log() assumes % formatting)

I just ignore logging's sugar. `.format` is definitely preferable.

This change LGTM.

File: gcloud/dns/changes.py [hunk 1]
```diff
@@ -221,7 +221,8 @@ def create(self, client=None):
         self._set_properties(api_response)
 
     def exists(self, client=None):
-        """API call:  test for the existence of the change set via a GET request
+        """API call:  test for the existence of the change set via a GET
+        request.
```
Comment (dhermes) [inline gcloud/dns/changes.py]: This is bad. Single-line descriptions are supposed to fit on a single line. So either change the wording or leave it as-is. The 80 char limit is also relaxed on docstrings.

File: gcloud/pubsub/connection.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 
 class Connection(base_connection.JSONConnection):
-    """A connection to Google Cloud Pubsub via the JSON REST API.
+    """A connection to Google Cloud Pub/Sub via the JSON REST API.
```
Comment (dhermes) [inline gcloud/pubsub/connection.py]: :+1:

File: gcloud/storage/batch.py [hunk 3]
```diff
@@ -232,7 +232,7 @@ def _finish_futures(self, responses):
             raise make_exception(*exception_args)
 
     def finish(self):
-        """Submit a single `multipart/mixed` request w/ deferred requests.
+        """Submit a single `multipart/mixed` request with deferred requests.
```
Comment (dhermes) [inline gcloud/storage/batch.py]: Why was this change necessary?

File: gcloud/resource_manager/client.py [hunk 4]
```diff
@@ -49,7 +49,7 @@ class Client(BaseClient):
     _connection_class = Connection
 
     def new_project(self, project_id, name=None, labels=None):
-        """Creates a :class:`.Project` bound to the current client.
+        """Create a :class:`.Project` bound to the current client.
```
Comment (dhermes) [inline gcloud/resource_manager/client.py]: Why was this necessary?

File: gcloud/dns/changes.py (hunk 1 above)
Comment (jonparrott) [inline gcloud/dns/changes.py]: It's up to you, but I really want to call out the fact that the 'single line description should be a single line' is broken *everywhere* in the Python community. Requests, urllib3, github3.py, etc. Sphinx has explicit support for the 'short description' being multiple lines.

File: gcloud/storage/batch.py (hunk 3 above)
Comment (waprin) [inline gcloud/storage/batch.py]: Better to avoid abbreviations of common words with international audiences.

**Comment by dhermes** [comment]
OK finished my pass and don't see many problems. I'd really like to hear the why on adding all the `.` but the typos are obvious so thanks.

**Comment by dhermes** [comment]
@waprin I see you made a new commit since I originally reviewed. Care to point me to what (in the 28 files) has changed?

File: gcloud/resource_manager/client.py (hunk 4 above)
Comment (waprin) [inline gcloud/resource_manager/client.py]: Repo owners need to decide whether english language usage is something you care about or not and consistently apply that decision. If you do, then consistent verb tense should be fair game.

Comment (dhermes) [inline gcloud/resource_manager/client.py]: > english language usage

Out of my depth there. Seems a real challenge to get lots of contributors on the same page with that (without someone dedicated to docs).

**Comment by waprin** [comment]
My bad I forgot to push commits again. I fixed the line that I moved to the next lien by removing an extraneous space. Will  remember to push commits. 

I don't know about the punctuation, depends on whether docstring consistency is important. Most of the repo uses periods at the end of each docstring param description.

File: gcloud/dns/changes.py (hunk 1 above)
Comment (dhermes) [inline gcloud/dns/changes.py]: But why exacerbate that brokenness if we don't have to?

**Comment by dhermes** [comment]
We're good to merge here / LGTM

I'm just trying to point out that human language is hard and enforcing these sorts of rules without a tool of some kind seems to be a big stretch goal.

**Comment by waprin** [comment]
Yes I agree 1000% that the more we can do with tools the better.

File: gcloud/dns/changes.py (hunk 1 above)
Comment (jonparrott) [inline gcloud/dns/changes.py]: (This particular case might be able to be reduced to one string, so just consider this a meta conversation)

Considering that the community has gone one way, I'd argue that PEP257 is what's broken. Practicality over dogma. The justification for it in PEP257 is

> The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line.

Considering that sphinx has long supported 'breaking' this rule that justification no longer seems totally relevant. I've written documentation for lots of projects and really the *most* important thing is getting your point across clearly, not hard and fast rules. If a slightly longer docstring helps someone understand what a method is *without having to read the entire docstring* then it's win. I definitely agree it should be *one sentence*.

Comment (tseaver) [inline gcloud/dns/changes.py]: @jonparrott ISTM that breaking with the PEP is just wrong: the "single line summary" discipline *works* as a way to focus intent.  When I find myself creeping up to the limit, finding a way to shorten that line (maybe by moving some content down to a more "narrative" paragraph) always improves it.

Comment (jonparrott) [inline gcloud/dns/changes.py]: Sure, as a general guideline it's fine. But it isn't so universally true to
be considered dogma. So many projects have rightfully broke it in order to
have better docs (flask and requests being notable examples).
